storybook/stories/API/cartesian/ReferenceLine.stories.tsx

```diff
@@ -130,6 +130,6 @@ export const API = {
     stroke: 'blue',
     strokeWidth: 2,
     strokeDasharray: '4 1',
-    label: { fill: 'red', fontSize: 20 },
+    label: 'My example label',
```

Comment:
Is this an unrelated fix? Do we need this change?

Reply:
Yep, the label is meant to be a text.